### PR TITLE
Described EditorConfig#initialData

### PR DESCRIPTION
**Pull request by scofalik**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Described `EditorConfig#initialData`. Closes https://github.com/ckeditor/ckeditor5/issues/1619.

---

### Additional information

This is a part of https://github.com/ckeditor/ckeditor5/issues/1619 together with PRs:

* https://github.com/ckeditor/ckeditor5-editor-decoupled/pull/30
* https://github.com/ckeditor/ckeditor5-editor-classic/pull/85
* https://github.com/ckeditor/ckeditor5-editor-inline/pull/50
* https://github.com/ckeditor/ckeditor5-editor-balloon/pull/31

I've added "Closes" only here.

### CODE REVIEW

File: src/editor/editorconfig.jsdoc [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * An object defining the editor configuration can be passed when initializing the editor:
  *
  *		EditorClass
- *			.create( {
+ *			.create( document.querySelector( '#editor' ), {
```
Comment (pjasiun) [inline src/editor/editorconfig.jsdoc]: This part of the documentation is not related to this ticket, and it needs bigger changes, so I think we should not touch it now.

File: src/editor/editorconfig.jsdoc [hunk 2]
```diff
@@ -30,6 +30,28 @@
  * @interface EditorConfig
  */
 
+/**
+ * The initial editor data to be used instead of the provided element's HTML content.
+ *
+ *      EditorClass
```
Comment (pjasiun) [inline src/editor/editorconfig.jsdoc]: You can use `ClassicEditor` here like some other parts of this docs.

File: src/editor/editorconfig.jsdoc (hunk 1 above)
Comment (scofalik) [inline src/editor/editorconfig.jsdoc]:
```suggestion
 *			.create( {
```

File: src/editor/editorconfig.jsdoc (hunk 2 above)
Comment (scofalik) [inline src/editor/editorconfig.jsdoc]:
```suggestion
 *      ClassicEditor
```

**Pull request merged**